src/checkers/ast.rs

```diff
@@ -4472,7 +4472,7 @@ impl<'a> Checker<'a> {
                     let multiple = unused_imports.len() > 1;
                     for (full_name, range) in unused_imports {
                         let mut diagnostic = Diagnostic::new(
-                            violations::UnusedImport(full_name.to_string(), ignore_init, multiple),
+                            violations::UnusedImport{full_name.to_string(), ignore_init, multiple, name: todo!() },
```

Comment:
Unfortunately this won't compile. Have you been able to set up a local development environment for Rust?

Reply:
Yeah I have actually but somehow when I run the tests it throws up 
``
Found 2806 error(s).
747 potentially fixable with the --fix option.
```

Reply:
@charliermarsh Yeah fixed the compile errors.